app/code/community/Bolt/Boltpay/Model/Order.php

```diff
@@ -659,8 +659,8 @@ protected function validateTotals(Mage_Sales_Model_Quote $immutableQuote, $trans
 
             if ( $cartItem->shouldNotBeValidated ){ continue; }
 
-            $boltPrice = (int)$boltCartItem->total_amount->amount;
-            $magentoRowPrice = (int) ( $cartItem->getRowTotalWithDiscount() * 100 );
+            $boltPrice = (int)round($boltCartItem->total_amount->amount);
+            $magentoRowPrice = (int)round( $cartItem->getRowTotalWithDiscount() * 100 );
```

Comment:
Is this necessary?  Does `$cartItem->getRowTotalWithDiscount()` return a value beyond two significant decimal digits?